config/routes.rb

```diff
@@ -1,10 +1,8 @@
 Rails.application.routes.draw do
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
 
-  get '/merchants/:merchant_id/dashboard', to: 'merchants#show', as: 'merchant_dashboard'
-
-  resources :merchants
-    resources :items
-    resources :invoices
-  end
+  get '/merchants/:merchant_id/items', to: 'items#index'
```

Comment:
The equivalent using resources would be 
```
resources :merchants do
  resources items, only: [:index, :show, :update, :edit]
end
  ``` 
  Or something similar if you want to play around with that syntax